packages/jest-core/src/cli/index.ts

```diff
@@ -276,9 +276,14 @@ const runWithoutWatch = async (
   filter?: Filter,
 ) => {
   const startRun = async (): Promise<void | null> => {
+    if (globalConfig.seed) {
+      outputStream.write(`Seed is ${globalConfig.seed}\n`);
```

Comment:
hmm, I wonder if it makes sense to have the reporters print this? in `onRunStart` or something.

Or maybe as part of `preRunMessagePrint`?

Reply:
It changes how the tests are run so I'd say it makes sense as part of `onRunStart`

The `preRunMessagePrint` seems like a temporary status which gets removed.

I'll see how easy it is to add the seed to `onRunStart`.